lib/cli/commands/DisableCooperative.ts

```diff
@@ -0,0 +1,26 @@
+import type { Arguments } from 'yargs';
+import { SetDisableCooperativeRequest } from '../../proto/boltzrpc_pb';
+import type { ApiType, BuilderTypes } from '../BuilderComponents';
+import { callback, loadBoltzClient } from '../Command';
+
+export const command = 'disablecooperative [disabled]';
```

Comment:
This command should ***never*** be used in production, so let's call this

```suggestion
export const command = 'dev-disablecooperative [disabled]';
```

And also adjust all mentions of it in the gRPC to have a `Dev` prefix